### ISSUE TITLE
Expose semantic annotations to web and other clients

### ISSUE THREAD

**Issue by amoeba** [NCEAS/metacatui#1655]
Semantic annotations in EML records make it possible to do some cool stuff like generating an RDF serialization for our EML records that triplifies the annotations for consumers. I'd like to see us expose this in some fashion though I'm not sure what it might look like just yet.

- We probably want to ensure EML annotations get into our JSON-LD exports at the very least
- We might think about how this relates and could integrate with the RDF data in our Data Packages
- What we do with enabling JSON-LD as a metadata format might impact this work. See https://github.com/DataONEorg/object-formats/issues/21 and https://github.com/DataONEorg/d1_cn_index_processor/issues/5

What do others think are some interesting opportunities here?

**Comment by mbjones** [comment]
I totally agree. It would be great to collect annotations from EML and provenance info from OREs and integrate it into the knowledge graphs, and then reflect it back in our JSON-LD exports. In general, I think many source should feed the graph, which can then be used to expose new information in our schema.org markup and our other indexed services like SOLR.
